### ISSUE TITLE
Getting started with AWS

### ISSUE THREAD

**Issue by alejandrox1** [Data4Democracy/tutorials#6]
Tutorial on `aws` and how to use `boto`.

**Comment by georgerichardson** [comment]
@zachmueller I remember somewhere seeing that you had something on this?

**Comment by zachmueller** [comment]
Yeah I'll try to get around to this soon... It's just been 12 hour days at the lab the last few weekends so it may be a few weeks before I have bandwidth to tackle this...

**Comment by bstarling** [comment]
I will give s3 tutorial a shot.
